pkg/util/usage/usage.go

```diff
@@ -85,7 +85,7 @@ func usage(mf *mainFlags, cfg *mimir.Config) error {
 		fmt.Fprint(fs.Output(), b.String(), "\n")
 	})
 
-	if !mf.printHelpAll {
+	if !printAll {
 		fmt.Fprintf(fs.Output(), "\nTo see all flags, use -help-all\n")
```

Comment:
Can we make the file descriptor an argument to the function here to enable easier testing? Or maybe return the `strings.Builder` or `string`?

Reply:
This PR basically just copies the functions from one place to the other, so I didn't want to perform any refactoring at the same time. I agree that there are improvements we can make here, in a follow-up PR maybe?